kratos/geometries/geometry.h

```diff
@@ -773,11 +775,35 @@ class Geometry : public PointerVector<TPointType>
          quality = VolumeToAverageEdgeLength();
        } else if(qualityCriteria == QualityCriteria::VOLUME_TO_RMS_EDGE_LENGTH) {
          quality = VolumeToRMSEdgeLength();
+       } else if(qualityCriteria == QualityCriteria::MIN_DIHEDRAL_ANGLE) {
```

Comment:
switch?

Reply:
I usually prefer the if else instead of the switch as the latter needs to put the break for correct working and from optimization point of view is not better than ifs